pyiron_base/jobs/job/runfunction.py

```diff
@@ -409,7 +409,7 @@ def run_job_with_runmode_queue(job):
     )
     if que_id is not None:
         job.server.queue_id = que_id
-        job._server.to_hdf(job._hdf5)
```

Comment:
This is what I worry about, maybe we can keep the hdf methods around for a bit and simply issue warnings in case they are called, since they are using the dict methods internally already anyway.

Reply:
From my perspective the `job.server` object is internal to `pyiron_base` so nobody should modify it. Still I am fine with adding a deprecation warning from `0.8` to `0.9`.

Reply:
I'm thinking more in case there are other places where we missed that this is still called.